src/components/blogCard/BlogCard.css

```diff
@@ -1,95 +1,140 @@
-.container {
+.blog-container{
+  border-radius: 10px;
+  color: #586069;
+  background-color: #fff;
+  box-shadow: 0 0 36px rgba(0,0,0,.1);
+  /* padding: 2rem; */
+  cursor: pointer;
+  height: 200px;
+}
+
+.blog-title {
+  color: #262626;
+  font-size: 22px;
+  line-height: 24px;
+  font-weight: 700;
+  margin-bottom: 4px;
+  padding: 10px 20px;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+}
+
+.small {
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  color: #666666;
+  padding: 0px 20px;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 4;
+  -webkit-box-orient: vertical;
+  margin: 0px;
+}
+
+.go-corner {
   display: flex;
+  align-items: center;
   justify-content: center;
+  position: absolute;
+  width: 32px;
+  height: 32px;
+  overflow: hidden;
+  top: 0;
+  right: 0;
+  background-color: #55198B;
+  border-radius: 0 4px 0 32px;
 }
 
-.square:hover {
-  box-shadow: 0 20px 40px rgba(0, 0, 0, 0.08);
+.go-arrow {
+  margin-top: -4px;
+  margin-right: -4px;
+  color: white;
+  font-family: courier, sans;
 }
 
-.square {
-  width: 440px;
-  height: 430px;
-  background: white;
+.blog-card {
+  display: block;
+  position: relative;
+  background-color: transparent;
   border-radius: 4px;
-  box-shadow: 0px 20px 50px #d9dbdf;
-  -webkit-transition: all 0.3s ease;
-  -o-transition: all 0.3s ease;
-  transition: all 0.3s ease;
-  margin-top: 30px;
+  text-decoration: none;
+  z-index: 0;
+  overflow: hidden;
+  height: 100%;
 }
 
-.mask {
-  clip: rect(0px, 460px, 220px, 0px);
-  border-radius: 4px;
-  position: absolute;
+.blog-card::before{
+    content: "";
+    position: absolute;
+    z-index: -1;
+    top: -16px;
+    right: -16px;
+    background: #55198B;
+    height: 32px;
+    width: 32px;
+    border-radius: 32px;
+    transform: scale(1);
+    transform-origin: 50% 50%;
+    transition: transform 0.40s ease-out;
 }
 
-.blog-card-image {
-  width: 440px;
+.blog-card:hover::before{
+  transform: scale(30);
 }
 
-.blog-card-title {
-  margin: auto;
-  text-align: left;
-  margin-top: 240px;
-  padding: 0 30px;
-  font-size: 24px;
-  overflow: hidden;
-  display: -webkit-box;
-  -webkit-line-clamp: 2;
-  -webkit-box-orient: vertical;
+ .blog-card:hover > .small {
+  transition: all 0.3s ease-out;
+  color: rgba(255, 255, 255, 0.8);
 }
-.blog-card-subtitle {
-  text-align: justify;
-  padding-left: 30px;
-  padding-right: 30px;
-  font-size: 12px;
-  color: #c8c8c8;
-  line-height: 18px;
-  overflow: hidden;
-  display: -webkit-box;
-  -webkit-line-clamp: 2;
-  -webkit-box-orient: vertical;
+
```

Comment:
Remove extra line, I'll enforce this with prettier soon but for now we have to do this manually.